integration/send-logs-gradient.py

```diff
@@ -52,14 +54,42 @@
 
 # COMMAND ----------
 
-response = databricks.record_run(
+response = databricks.create_submission_for_run(
         run_id=dbutils.widgets.get("DATABRICKS_RUN_ID") or dbutils.widgets.get("DATABRICKS_PARENT_RUN_ID"),
         plan_type=dbutils.widgets.get("DATABRICKS_PLAN_TYPE"),
         compute_type=dbutils.widgets.get("DATABRICKS_COMPUTE_TYPE"),
         project_id=dbutils.widgets.get("SYNC_PROJECT_ID"),
         exclude_tasks=([dbutils.widgets.get("DATABRICKS_TASK_KEY")] if dbutils.widgets.get("DATABRICKS_TASK_KEY") else None),
-        allow_incomplete_cluster_report=True)
+        allow_incomplete_cluster_report=True,
+    )
 
-print(response)
+if response.error:
+    raise RuntimeError(str(response.error))
+
+response = projects.get_project(dbutils.widgets.get("SYNC_PROJECT_ID"))
+
+if response.error:
+    raise RuntimeError(str(response.error))
+
+project = response.result
+if project.get("auto_apply_recs"):
+    response = projects.create_project_recommendation(project["id"])
+
+    if response.error:
+        raise RuntimeError(str(response.error))
+
+    recommendation_id = response.result
+
+    response = projects.wait_for_recommendation(project["id"], recommendation_id)
+
+    if response.error:
+        raise RuntimeError(str(response.error))
```

Comment:
Do we not want to combine these operations into one method in the library? You might just be saving that as followup work.